client/python/lib/ovmsclient/tfs_compat/grpc/requests.py

```diff
@@ -144,5 +148,24 @@ def make_status_request(model_name, model_version=0):
         >>> print(status_request)
 
     '''
+
+    _check_model_spec(model_name, model_version)
+
+    request = get_model_status_pb2.GetModelStatusRequest()
+    request.model_spec.name = model_name
+    request.model_spec.version.value = model_version
+    return GrpcModelStatusRequest(model_name, model_version, request)
+
+def _check_model_spec(model_name, model_version):
+
+    if not isinstance(model_name, str):
+        raise TypeError(f'model_name should be type string, but is type {type(model_name).__name__}')
     
-    raise NotImplementedError
+    if not isinstance(model_version, int):
+        raise TypeError(f'model_version should be type int, but is type {type(model_version).__name__}')
+
+    if not model_version.bit_length() <= 63:
+        raise ValueError(f'model_version should have max 63 bits, but has {model_version.bit_length()}')
```

Comment:
I would change this message to be easier for the user to interpret like:
`model_version should be in range <0, 9223372036854775807>`
In the code, upper bound should be evaluated (like: 2^63-1) rather than hardcoded.